app/javascript/css/forms.scss

```diff
@@ -217,6 +217,10 @@ input[type='date']:invalid::-webkit-datetime-edit {
   overflow: hidden;
 }
 
+.income-field {
```

Comment:
is this needed? we should be able to accomplish width constraints using bootstrap flex

Reply:
Bootstrap flex options do not split the row 50/50 no matter which flex class is used, easiest solution was to introduce a new class

Reply:
I think d-flex and cols can be used exactly for this purpose. I just tried on my local and it works. May need some extra work to get smaller breakpoints to play nice, but not a huge deal there probably

Reply:
removed after refactor